contracts/cosmwasm-vm/cw-ibc-core/tests/channel/test_receive_packet.rs

```diff
@@ -283,67 +282,8 @@ fn execute_receive_packet_ordered() {
         .unwrap();
 
     let res = contract.execute_receive_packet(deps.as_mut(), reply);
-    let seq = contract.get_next_sequence_recv(
-        &deps.storage,
-        &IbcPortId::from_str(&packet.dest.port_id).unwrap(),
-        &IbcChannelId::from_str(&packet.dest.channel_id).unwrap(),
-    );
-    assert!(res.is_ok());
-    assert_eq!(res.unwrap().events[0].ty, "recv_packet");
-    assert!(seq.is_ok());
-    assert_eq!(seq.unwrap(), 2.into())
-}
-#[test]
-#[should_panic(
-    expected = "Std(NotFound { kind: \"common::ibc::core::ics04_channel::packet::Sequence\" })"
-)]
-fn execute_receive_packet_ordered_fail_missing_seq_on_a() {
-    let contract = CwIbcCoreContext::default();
```

Comment:
This error was being tested on reply when called before initializing counters but didn't make sense since the logic was moved to main function from reply.

Reply:
Just wondering if we need to add tests to the main function to cover the new error cases there

Reply:
ya i will have separate task for that for packet flow